lib/spack/spack/build_environment.py

```diff
@@ -886,7 +886,7 @@ def child_process(child_pipe, input_stream):
         if sys.stdin.isatty() and hasattr(sys.stdin, 'fileno'):
             input_stream = os.fdopen(os.dup(sys.stdin.fileno()))
 
-        p = multiprocessing.Process(
+        p = fork_context.Process(
```

Comment:
This one is definitely needed, otherwise you can't install anything:
```
AttributeError: Can't pickle local object 'fork.<locals>.child_process'
```